wiki/History_of_osu!/April_Fools/en.md

```diff
@@ -0,0 +1,324 @@
+
+### Bad Apple ranking chart
+
+The "Bad Apple Ranking Chart" was a joke ranking chart based off of a variety of [beatmaps](/wiki/Beatmap) that featured the song "Bad Apple!!" as part of osu!'s 2012 April Fools' joke. Announced via [a newspost](https://osu.ppy.sh/community/forums/topics/79128?start=1431905) on 1 April 2012, the ranking chart actually functioned as a real working chart at the time and displayed the top 40 players who obtained the highest [ranked score](/wiki/Score#ranked-score) in any one of the handpicked selection of "Bad Apple!!" beatmaps \[1,2,3\]. An excerpt from the aforementioned newspost is inscribed below:
+
+> We decided to dedicate this chart to the greatest song and video ever created, Bad Apple!!. You can find the awesome chart here
+>
+> Now, as this is a chart of epic proportions, we needed to make our prize a little better this time! The winners will receive a Lily White poster and a hand drawn picture of Reimu. (A printer is required to receive these prizes)
+>
+> Look forward to our next charts, the Renai Circulation and Irony charts!
+
+— Cyclone, "Bad Apple Ranking Chart!" \[2,3\]
+
+The chart was opened on 1 April 2012 and closed on 2 April 2012. At the end of the ranking period, player [Mesita](https://osu.ppy.sh/users/201459) was the number one player with a ranked score of 145,623,328 \[4\].
+
+The beatmaps included in the ranking chart are listed below:
+
+- [nomico - Bad Apple!! (James)](https://osu.ppy.sh/beatmapsets/6252)
+- [REDALiCE - Bad Apple!! (Rena-chan)](https://osu.ppy.sh/beatmapsets/10353)
+- [Masayoshi Minoshima ft. nomico - Bad Apple!! (Ephemeral)](https://osu.ppy.sh/beatmapsets/10435)
+- [Masayoshi Minoshima ft. nomico - Bad Apple!! (ignorethis)](https://osu.ppy.sh/beatmapsets/13177)
+- [Masayoshi Minoshima feat. StrawbellyCake - Bad Apple!! (German Version) (Larto)](https://osu.ppy.sh/beatmapsets/13664)
+- [Masayoshi Minoshima feat. Larto & nomico - Awesome Apple!! (Larto)](https://osu.ppy.sh/beatmapsets/14475)
+- [Masayoshi Minoshima feat. nomico - Bad Apple!! (ouranhshc)](https://osu.ppy.sh/beatmapsets/18260)
+- [Spiritsoulxx - Bad Apple!! (Tony)](https://osu.ppy.sh/beatmapsets/23760)
+- [Kommisar - Bad Apple!! (Chiptune Ver.) (Sushi)](https://osu.ppy.sh/beatmapsets/28222)
+- [Kalafina - Bad MagiApple (Makar8000)](https://osu.ppy.sh/beatmapsets/32003)
+
+The "Bad Apple!!" music video had been some sort of an in-joke at the time with remixes of the song combined with other ironic twists being a prevalent occurrence in beatmaps at the time, hence its appearance as an April Fools' joke. <!--citation needed-->
+
+#### References
+
+1. [Bad Apple!! Ranking Chart, 4 April 2012](https://web.archive.org/web/20120404051938/http://osu.ppy.sh/p/chart?ch=BADAPPLE "Internet Archive — osu!")
+2. [osu! newspost [Cyclone] (1 April 2012) - "Bad Apple!! Ranking Chart"](https://osu.ppy.sh/community/forums/topics/79128?start=1431905)
```

Comment:
both of these are admin forbidden, but if im not mistaken isn't 1431905 the number attributed to start and not the post number?

Reply:
that is the permalink that goes to the forum post specifically, rather than the topic. both redirect to the same page and same post, but my suggestion contains a clearer one